client/src/views/Settings/Tabs/EditProfile/index.vue

```diff
@@ -422,11 +442,11 @@ export default {
         })
       }
       this.profile = { ...this.emptyProfile, ...this.profileGetter, socialAccounts }
-      if (this.profile.avatar_url && JSON.stringify(this.profile.avatar_url) === JSON.stringify({})) {
-        this.profile.avatar_url = ""
+      if (this.profile.avatarUrl && JSON.stringify(this.profile.avatarUrl) === JSON.stringify({})) {
```

Comment:
Please create utility method to validate avatarUrl and backgroundUrl and simply if(utilityMethod(this.profileAvatarUrl)).